### DISCUSSION TITLE
cannot receive email when custom event shows up

### DISCUSSION THREAD

**Discussion by drgnchan** [community/community#42640] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

1. Setting: Notify me by email when specified event emits:

<img width="823" alt="image" src="https://user-images.githubusercontent.com/40224023/209492757-eaf57df4-9c7d-4617-9978-bba5b450beb5.png">

2. I have subscribed skywalking:

<img width="806" alt="image" src="https://user-images.githubusercontent.com/40224023/209492865-2098e8bb-165e-4cb1-bb34-704bb67638c3.png">

3. there is a notification in box:

<img width="982" alt="image" src="https://user-images.githubusercontent.com/40224023/209492898-dc2863b6-d86e-4377-a0e8-d879aee54b73.png">

4. but i cannot receive any email.

**Comment by radar-blip** [comment]
Since you are watching that repository (apache/skywalking) 
I think you would have to set your watching settings to recieve emails also.

![watching](https://user-images.githubusercontent.com/39196767/209508191-5b6cb363-8135-44f6-b654-e3e67372def0.png)

**Reply by drgnchan** [reply]
> Since you are watching that repository (apache/skywalking) I think you would have to set your watching settings to recieve emails also.
> 
> ![watching](https://user-images.githubusercontent.com/39196767/209508191-5b6cb363-8135-44f6-b654-e3e67372def0.png)

But I dont want to receive email from all watching repositories.

**Reply by radar-blip** [reply]
Sorry, I dont know how you could do that.

**Reply by drgnchan** [reply]
Thank you!
